io.kavan.GUADEC.json

```diff
@@ -0,0 +1,77 @@
+{
+    "app-id": "io.kavan.GUADEC",
+    "runtime": "org.gnome.Sdk",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "guadec-events",
+     "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
```

Comment:
We do not allow Internet access during build; you can generate a manifest for dependencies with https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo.